karma.conf.js

```diff
@@ -94,6 +109,8 @@ function combinedLoaders() {
     case 'istanbulInstrumenter':
     case 'tslint':
       return aggregate;
+    case 'ts':
```

Comment:
Good catch. Though the current setup isn't failing, removing the redundant `ts` loader does speed things up a bit. I would just change it to:
```
case 'tslint':
case 'ts':
  return aggregate;
```